frontend/src/__tests__/cypress/cypress/tests/mocked/pipelines/pipelinesList.cy.ts

```diff
@@ -163,7 +163,8 @@ describe('PipelinesList', () => {
     pipelineRow
       .getPipelineVersionRowById(initialMockPipelineVersion.pipeline_version_id)
       .findPipelineVersionLink()
-      .click();
+      // TODO - remove force one https://github.com/patternfly/patternfly-react/issues/11314 is fixed
+      .click({ force: true });
```

Comment:
With the css override for table truncation, I think we can remove this workaround.
```suggestion
      .click();
```